Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -20,20 +20,14 @@
             HOMEBREW_LINUX_DEFAULT_PREFIX].include?(prefix) %>
     prefix "<%= prefix %>"
     <% end %>
-    <% if cellar.is_a? Symbol %>
```

Comment:
I think it's worth keeping this as-is for the first PR and move this logic into a later one. Until then, bottle blocks can be hand-crafted for testing.

Reply:
Agreed. I pushed a new version where I am not modifying the writing part at all.